package/wpe/wpeframework-plugins/wpeframework-plugins.mk

```diff
@@ -72,6 +72,17 @@ ifeq ($(BR2_PACKAGE_WPEFRAMEWORK_DICTIONARY),y)
 WPEFRAMEWORK_PLUGINS_CONF_OPTS += -DPLUGIN_DICTIONARY=ON
 endif
 
+ifeq ($(BR2_PACKAGE_WPEFRAMEWORK_SECURESHELLSERVER),y)
```

Comment:
Can you move this down to other S related selections? Trying to organize this A-Z for readability.

Reply:
Yes. Corrected.